crates/js-sys/tests/wasm/Symbol.rs

```diff
@@ -107,3 +107,11 @@ fn value_of() {
     let a = gensym(JsValue::undefined());
     assert_eq!(JsValue::from(a.value_of()), JsValue::from(a));
 }
+
+#[wasm_bindgen_test]
+fn symbol_inheritance() {
```

Comment:
And I think this was accidentally included?

Reply:
No, It wasn't accidentally. Symbol is like other, isn't it ?
It has 2 Constructor which it inherit from : Symbol and Object